superset-frontend/src/views/CRUD/data/dataset/AddDataset/EditDataset/index.tsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { styled, SupersetClient, t, logging } from '@superset-ui/core';
+import React, { useEffect, useState } from 'react';
+import Badge from 'src/components/Badge';
+import Tabs from 'src/components/Tabs';
+
+const StyledTabs = styled(Tabs)`
+  ${({ theme }) => `
+  margin-top: ${theme.gridUnit * 8.5}px;
+  padding-left: ${theme.gridUnit * 4}px;
+
+  .ant-tabs-top > .ant-tabs-nav::before {
+    width: ${theme.gridUnit * 50}px;
+  }
+  `}
+`;
+
+const TabStyles = styled.div`
+  ${({ theme }) => `
+  .ant-badge {
+    width: ${theme.gridUnit * 8}px;
+    margin-left: ${theme.gridUnit * 2.5}px;
+  }
+  `}
+`;
+
+interface EditPageProps {
+  id: string;
+}
+
+const TRANSLATiONS = {
```

Comment:
lowercase `i`?

Reply:
Whoops! Got cause using the shift key instead of caps lock 🤣 
Fixed in [`this commit`](https://github.com/apache/superset/pull/22043/commits/77f5c7f40fe0c0269d5ef8a2f3e6ad2483d8e180).